### PR TITLE
[tests-only][full-ci] add test coverage for list permission for non-member user in project-space

### PR DESCRIPTION
**Pull request by nabim777**
## Description

Here following test coverage scenario is added for the issue-9151

```feature
Scenario: non-member user tries to list the permissions of a project space using root endpoint
```

Also, there is added of issue tag which was removed in the PR https://github.com/owncloud/ocis/pull/9331/files

## Related Issue

- Related: https://github.com/owncloud/ocis/issues/8428 and https://github.com/owncloud/ocis/issues/9151

## How Has This Been Tested?

- test environment:
- ci
- locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by saw-jan — APPROVED** [review]

File: tests/acceptance/features/apiSharingNg/listPermissions.feature [hunk 1]
```diff
@@ -1696,3 +1696,45 @@ Feature: List a sharing permissions
       | space     | new-space |
       | sharee    | Brian     |
       | shareType | user      |
+
+
+  @issue-9151
```
Comment (saw-jan) [inline tests/acceptance/features/apiSharingNg/listPermissions.feature:1701]:
```suggestion

  @issue-9151
```

**Review by saw-jan — CHANGES_REQUESTED** [review]

File: tests/acceptance/features/apiSharingNg/listPermissions.feature [hunk 2]
```diff
@@ -1696,3 +1696,45 @@ Feature: List a sharing permissions
...
+  Scenario: non-member user tries to list the permissions of a project space using permissions root endpoint
```
Comment (saw-jan) [inline tests/acceptance/features/apiSharingNg/listPermissions.feature:1702]: Please, add test with permissions endpoint as well

Comment (saw-jan) [inline tests/acceptance/features/apiSharingNg/listPermissions.feature:1702]:
```suggestion
  Scenario: non-member user tries to list the permissions of a project space using root endpoint
```

Comment (nabim777) [reply tests/acceptance/features/apiSharingNg/listPermissions.feature:1701]: Isn't that test already covered?
https://github.com/owncloud/ocis/pull/9423/files#diff-77314df4d252124f66502910bdfc4343a87d01f39299398d45182c1d1ca5caedL1290

**Review by saw-jan — APPROVED** [review]

**Pull request merged**